controllers/history.controller.js

```diff
@@ -0,0 +1,107 @@
+const User = require("../models/user.model");
+const History = require("../models/history.model");
+const { concat, remove } = require("lodash");
+
+const getHistory = async (req, res) => {
+  const history = await History.find({});
+  res.json({ success: true, history });
+};
+
+const findUserHistory = async (req, res, next, userId) => {
+  try {
+    let user = await User.findOne({ _id: userId });
+    if (!user) {
+      res.status(404).json({
+        success: false,
+        message: "Invalid user! Kindly register to continue",
+      });
+      throw Error("Invalid User");
+    }
```

Comment:
Hey I have one doubt here, why do you have to throw Error if you are sending response in line number 14, a return statement should work here.